data_loading.py

```diff
@@ -1915,6 +1923,47 @@ def process_uproot_numu(up, df):
 
     return
 
+def drop_vector_columns(df):
```

Comment:
There is possibly a conflict here: I removed the vector columns at their source, i.e. commented them out from being loaded entirely, and then deleted this function because it is no longer necessary.

Reply:
Aren't some of those vector columns required to make the non vector columns though?

Reply:
They are always used in a self-contained way in the `process_` functions, for example 
```python
def process_uproot_ccncpi0vars(up, df):
    mc_pdg = up.array("mc_pdg")
    mc_E = up.array("mc_E")
    mc_px = up.array("mc_px")
    mc_py = up.array("mc_py")
    mc_pz = up.array("mc_pz")
```
loads the `mc_pdg` rugged array in a variable that is used within the function, but goes out of scope after the function and is never written into the dataframe.

Reply:
I think not loading them is fine then.